examples/context/context.sh

```diff
@@ -1,5 +1,5 @@
 # Run the server in the background.
-$ go run context-in-http-servers.go &
+$ go run context.go &
```

Comment:
This change is worthwhile, but the http-server(s) http-client(s) - I prefer to keep as is

Reply:
gotcha. Just reverted those changes.